cli/azd/cmd/build.go

```diff
@@ -198,7 +200,7 @@ func (ba *buildAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 
 	return &actions.ActionResult{
 		Message: &actions.ResultMessage{
-			Header: "Your Azure app has been built!",
+			Header: fmt.Sprintf("Your Azure app was built in %s.", ux.DurationAsText(time.Since(startTime))),
```

Comment:
A nit but I feel like I'd like it better if it were

"Your application was [ _action_] for Azure in [ _time_]"

Technically, the app exists outside the context of Azure...we're just helping the dev get it on Azure.

Reply:
How does

azd build: "Your application was built for Azure in ... seconds."
azd deploy: "Your application was deployed to Azure in ... seconds."
azd down: "Your application was deleted from Azure in ... seconds."
azd package: "Your application was packaged for Azure in ... seconds."
azd provision: "Your application was provisioned in Azure in ... seconds."
azd restore: "Your application's dependencies were restored in ... seconds."
azd up: "Your application was provisioned and deployed to Azure in ... seconds."

Read to the room?

Reply:
I'll go with "removed from".